src/Tasks/Microsoft.NET.Build.Tasks/ResolveRuntimePackAssets.cs

```diff
@@ -71,6 +71,7 @@ void AddAsset(string assetPath, string assetType)
                         assetPath.EndsWith(".map", StringComparison.OrdinalIgnoreCase) ||
                         assetPath.EndsWith(".txt", StringComparison.OrdinalIgnoreCase) ||
                         assetPath.EndsWith(".xml", StringComparison.OrdinalIgnoreCase) ||
+                        assetPath.EndsWith(".json", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
Should this be an include list instead of exclude list?

Reply:
The reason it's not an include list is because native assets can have a variety of extensions and I'm not sure if we know what that list is.

Eventually we would like the runtime packs to [include a manifest](https://github.com/dotnet/cli/issues/10442#issuecomment-454587259) which lists the managed and native files in the pack explicitly, so we don't have to glob for them at all.

Reply:
https://github.com/dotnet/cli/issues/10537 tracks the manifests.

Reply:
> native assets can have a variety of extensions and I'm not sure if we know what that list is.

Can't we look at the fixed set of runtime packs we have now? Is it not .dll,.exe,.so,.dylib?

Switching to include list until we have the manifests seems reasonable to me to err on the safer side.

Reply:
But I'm ok with this more targeted fix to unblock things with the least change given that we have an even safer medium term plan.